program-runtime/src/invoke_context.rs

```diff
@@ -751,10 +758,20 @@ impl<'a> InvokeContext<'a> {
             }
         } else {
             // Verify the calling program hasn't misbehaved
-            self.verify_and_update(instruction_accounts, true)?;
+            let mut verify_caller_time = Measure::start("verify_caller_time");
+            let verify_caller_result = self.verify_and_update(instruction_accounts, true);
+            verify_caller_time.stop();
+            timings
```

Comment:
These are cumbersome, would be helpful if there were helpers here, like `timings.accumulate_ process_instruction_verify_caller_us(verify_caller_time.as_us())`

Reply:
I'll add a `saturating_add_assign!()` macro, as suggested by Alex.  We can easily replace it once [`std::num::Saturating`](https://doc.rust-lang.org/std/num/struct.Saturating.html) is stabilized.

Reply:
Done